internal/keepers/factory.go

```diff
@@ -1,3 +1,8 @@
+/*
+The keepers package is deprecated and will no longer be expected to function. It
+will be kept for a short period of time for reference. Use functions in the
```

Comment:
Not sure the value of keeping this here. It will always be present in git history.

imo would be just cleaner to delete it to prevent confusion / accidental use